<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AngpInputBox</class>
 <widget class="QWidget" name="AngpInputBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AngpInputBox</string>
  </property>
  <widget class="QGroupBox" name="parametersBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>79</height>
    </rect>
   </property>
   <property name="title">
    <string>Дополнительные параметры</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="dist_to_target_label">
      <property name="text">
       <string>Дистанция до цели</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="distToTargetBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0)</string>
      </property>
      <property name="minimum">
       <number>-1000000000</number>
      </property>
      <property name="maximum">
       <number>1000000000</number>
      </property>
      <property name="singleStep">
       <number>1000</number>
      </property>
      <property name="value">
       <number>15000</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="q_zad_label">
      <property name="text">
       <string>Заданный ракурс</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSpinBox" name="q_zadBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0)</string>
      </property>
      <property name="minimum">
       <number>-360</number>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="singleStep">
       <number>30</number>
      </property>
      <property name="value">
       <number>180</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="gamma1_label">
      <property name="text">
       <string>Крен разворота 1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="gamma1Box">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0)</string>
      </property>
      <property name="minimum">
       <number>-75</number>
      </property>
      <property name="maximum">
       <number>75</number>
      </property>
      <property name="singleStep">
       <number>15</number>
      </property>
      <property name="value">
       <number>45</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="gamma2_label">
      <property name="text">
       <string>Крен разворота 2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="gamma2Box">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0)</string>
      </property>
      <property name="minimum">
       <number>-75</number>
      </property>
      <property name="maximum">
       <number>75</number>
      </property>
      <property name="singleStep">
       <number>15</number>
      </property>
      <property name="value">
       <number>-60</number>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>gamma2Box</zorder>
   <zorder>gamma1_label</zorder>
   <zorder>q_zadBox</zorder>
   <zorder>gamma1Box</zorder>
   <zorder>dist_to_target_label</zorder>
   <zorder>gamma2_label</zorder>
   <zorder>distToTargetBox</zorder>
   <zorder>q_zad_label</zorder>
  </widget>
  <widget class="QGroupBox" name="manevrBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Метод наведения</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="2">
     <widget class="QRadioButton" name="pogonya">
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Погоня</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QRadioButton" name="manevr">
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Маневр</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="perh_manev">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Перехват с маневром</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="perehvat">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Перехват</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>pogonya</zorder>
   <zorder>perehvat</zorder>
   <zorder>perh_manev</zorder>
   <zorder>manevr</zorder>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Пуск</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="optimal_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Поиск оптимальных виражей</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
